system_test/m1_facilitator/SystemTest.ts

```diff
@@ -12,17 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import DepositSystemTest from './DepositSystemTest';
-// import WithdrawSystemTest from './WithdrawSystemTest';
+// import DepositSystemTest from './DepositSystemTest';
+import WithdrawSystemTest from './WithdrawSystemTest';
 import Logger from '../../src/common/Logger';
 
 async function runSystemTest(): Promise<void> {
   Logger.info('Running deposit and withdraw system test');
 
 
   await Promise.all([
-    DepositSystemTest.run(),
-    // WithdrawSystemTest.withdrawSystemTest(),
+    // DepositSystemTest.run(),
```

Comment:
Should this be uncommented? 🤔

Reply:
For testing on local, i did so as we were working on same branch.
Currently testing the entire system tests on local and reverting this change.